tests/search/search-test.spec.ts

```diff
@@ -0,0 +1,210 @@
+    expect(await desktopSortSelector.count()).toEqual(1);
+
+    expect(await expect(mobileSort).toHaveClass(/hidden/));
+    expect(await expect(desktopSort).toHaveClass(/visible/));
+
+    // sort-bar-filter URL changes
+    await page.getByRole('button', { name: 'Relevance' }).click();
+
+    await page.getByText('Weekly views').first().click();
+    await expect(page).toHaveURL(/query=cats&sort=-week/);
+
+    await page.getByRole('button', { name: 'Toggle options Weekly views' }).getByRole('button').click();
+    await page.getByRole('button', { name: 'All-time views' }).click();
+    await expect(page).toHaveURL(/query=cats&sort=-downloads/);
+
+    await page.getByRole('button', { name: 'Title' }).click();
+    await expect(page).toHaveURL(/query=cats&sort=title/);
+
+    await page.getByText('Date published').first().click();
+    await expect(page).toHaveURL(/query=cats&sort=-date/);
+
+    await page.getByRole('button', { name: 'Toggle options Date published' }).getByRole('button').click();
+    await page.getByRole('button', { name: 'Date archived' }).click();
+    await expect(page).toHaveURL(/query=cats&sort=-publicdate/);
+ 
+    await page.getByRole('button', { name: 'Toggle options Date archived' }).getByRole('button').click();
+    await page.getByRole('button', { name: 'Date reviewed' }).click();
+    await expect(page).toHaveURL(/query=cats&sort=-reviewdate/);
+
+    await page.getByRole('button', { name: 'Toggle options Date reviewed' }).getByRole('button').click();
+    await page.getByRole('button', { name: 'Date added' }).click();
+    await expect(page).toHaveURL(/query=cats&sort=-addeddate/);
+
+    await page.getByRole('button', { name: 'Creator' }).click();
+    await expect(page).toHaveURL(/query=cats&sort=creator/);
+  });
+  
+  test('collection-search-input options', async () => {
+    const collectionSearchInput = page.locator('collection-search-input');
+    const collectionSearchGo = collectionSearchInput.locator('#go-button');
+    const buttonCollapserLocator = collectionSearchInput.locator('#button-collapser');
+
+    expect(await collectionSearchInput.count()).toEqual(1);
+    expect(await collectionSearchGo.count()).toEqual(1);
+
+    const searchMetadata = buttonCollapserLocator.getByText('Search metadata');
+    const searchTextContents = buttonCollapserLocator.getByText('Search text contents');
+    const searchTVCaptions = buttonCollapserLocator.getByText('Search TV news captions');
+    const searchRadioTranscripts = buttonCollapserLocator.getByText('Search radio transcripts');
+    const searchArchivedWebSites = buttonCollapserLocator.getByText('Search archived web sites');
+    
+    expect(await searchMetadata.count()).toEqual(1);
+    expect(await searchTextContents.count()).toEqual(1);
+    expect(await searchTVCaptions.count()).toEqual(1);
+    expect(await searchRadioTranscripts.count()).toEqual(1);
+    expect(await searchArchivedWebSites.count()).toEqual(1);
```

Comment:
these are ok tests, they check the value and existence but do not check for behaviour? how can we test them to see results working?